File: ios/customNotification.m
//
//  customNotification.m
//  notification2
//
//  Created by Abdulloh Olimov on 21/09/24.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridge.h>

@interface RCT_EXTERN_MODULE(customNotification, NSObject)

RCT_EXTERN_METHOD(MyNotification:(NSString *)param)
@end

File: ios/customNotification.swift
//
//  customNotification.swift
//  notification2
//
//  Created by Abdulloh Olimov on 21/09/24.
//

//import Foundation

//@objc(customNotification) class  customNotification:NSObject{
//  @objc static func requireMainQueueSetup() ->Bool { return true}
//  
//  @objc public func MyNotification(param:String){
//    print(param)
//  }
//}

import Foundation

@objc(customNotification) class customNotification: NSObject {
  @objc static func requireMainQueueSetup() -> Bool {
    return true
  }

  @objc public func MyNotification(_ param: String) {
    print(param + "Hello")
  }
}

